### DISCUSSION TITLE
docs/basic/try-with-docker/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#3027] [Docs comments]
# docs/basic/try-with-docker/

Try Casdoor with Docker

https://casdoor.org/docs/basic/try-with-docker/

**Comment by INIGHub** [comment]
Under option 3 the page suggests looking into the docker-compose.yml file, which is not provided on this page itself.  For those who may be looking for it the reference is here: 

https://github.com/casdoor/casdoor/blob/master/docker-compose.yml
